packages/connection-storage/src/connection-storage.spec.ts

```diff
@@ -285,55 +189,29 @@ describe('ConnectionStorage', function () {
       expect(error.message).to.be.equal('id must be a uuid');
     });
 
-    it('does not save fle secrets if fleOptions.storeCredentials is false', async function () {
```

Comment:
Moved these fle tests to `connection-secrets.spec.ts` as they tested `extractSecrets` and were not relevant here